src/main/java/com/g4s8/ghman/bot/BotApp.java

```diff
@@ -106,7 +107,7 @@ public void run() {
                                 new TkCloseIssue(this.users)
                             )
                         ),
-                        new FbUnauthorized()
+                        new FbUnauthorized(new EnvironmentVariables())
```

Comment:
@olenagerasimova `BotApp` should be the one taking an `EnvironmentVariables` object and passing it to `FbUnauthorized` as well as other objects if needed.